python/utilities/MixingTimeResolution_Aux.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include <pybind11/pybind11.h>
+
+#include <goofit/PDFs/physics/MixingTimeResolution_Aux.h>
+
+using namespace GooFit;
+namespace py = pybind11;
+
```

Comment:
Have you tried running `pytest` in the build directory, and have you made sure you are using the version you built? You might need to run `pip -v install . -e` in the main directory, depending on which build python is picking up. You can use `pip uninstall goofit` to make sure you don't pick up the pip build.